cmd/tracee-rules/main.go

```diff
@@ -82,15 +83,20 @@ func main() {
 			}
 
 			var loadedSigIDs []string
-			for _, s := range sigs {
-				m, err := s.GetMetadata()
-				if err != nil {
-					logger.Error("failed to load signature", "error", err)
-					continue
-				}
-				loadedSigIDs = append(loadedSigIDs, m.ID)
-			}
-
+			capabilities.GetInstance().Requested(
+				func() error {
+					for _, s := range sigs {
+						m, err := s.GetMetadata()
+						if err != nil {
+							logger.Error("failed to load signature", "error", err)
+							continue
+						}
+						loadedSigIDs = append(loadedSigIDs, m.ID)
+					}
+					return nil
```

Comment:
Yep, its right. We can fail to load one signature but be able to load the other, thus the logging error instead of fully failing. Note this was already the behavior, I didn't change it:

```go
for _, s := range sigs {
        m, err := s.GetMetadata()
        if err != nil {
                logger.Error("failed to load signature", "error", err)
                continue
        }
        loadedSigIDs = append(loadedSigIDs, m.ID)
}
```

Reply:
done ?